donut/modules/groups/routes.py

```diff
@@ -18,6 +18,10 @@ def get_groups_list():
         fields = [f.strip() for f in flask.request.args["fields"].split(',')]
     return jsonify(helpers.get_group_list_data(fields=fields, attrs=attrs))
 
+@blueprint.route("/1/positions/")
+def get_positions():
+    return json.dumps(helpers.get_position_data())
```

Comment:
Can use `jsonify` here; takes care of setting the `Content-Type: application/json` for you